### PR TITLE
Updated example test cases to show error in test application

### PR DESCRIPTION
**Pull request by GeoGegl**
Added a test case for the Keyboard Down and Up Events labels in the test application to show an error in it. Extended existing keywords "Press key" and "Press keys" to include press and release events to enable the test.

In the test application "WpfApplication.exe" in the "Keyboard Controls" tab, the Keyboard Down Events label does not work for some keys. E.g. Arrow Keys (The Keyboard Up Events work)

### CODE REVIEW

**Review by noubar — CHANGES_REQUESTED** [review]

File: atests/Keyboard.robot [hunk 1]
```diff
@@ -143,9 +146,36 @@ False Argument Type
...
+
+
 *** Keywords ***
 Reset Textbox
     Click    ${XPATH_RESET}
     ${TEXT}  Get Name From Element  ${XPATH_LABEL_INPUT_UP}
     Should Be Empty  ${TEXT}
     Set Text To Textbox  ${XPATH_INPUT_FIELD}  ${EMPTY}
+
+Check Key Up Key Down
```
Comment (Nepitwin) [inline atests/Keyboard.robot:164]: Can you pls adjust spaces. Too many spaces are used. Pls set to arguments.

File: atests/Keyboard.robot [hunk 2]
```diff
@@ -143,9 +146,36 @@ False Argument Type
 	${ERR_MSG}      Run Keyword and Expect Error   *  Press Key   ${KEYBOARD_INPUT_TEXT_ARRAY}  ${XPATH_INPUT_FIELD}
     Should Be Equal As Strings  ${EXP_ERR_MSG}  ${ERR_MSG}
 
+Keyboard Key Down Key Up
+    [Documentation]    Check if Key Down / Key Up Events are displayed correctly
+	...                in their respective labels
+	FOR    ${KEY}    IN     @{KEYBOARD_KEY_LIST}
```
Comment (Nepitwin) [inline atests/Keyboard.robot:152]: Can you pls adjust spaces. Too many spaces are used. Pls set to arguments.

File: atests/Keyboard.robot [hunk 3]
```diff
@@ -20,12 +20,15 @@ Test Setup       Wait Until Keyword Succeeds    5x    10ms    Reset Textbox
 *** Variables ***
 ${XPATH_INPUT_FIELD}        ${MAIN_WINDOW_KEYBOARD_CONTROLS}/Edit[@AutomationId='KeyboardInputField']
 ${XPATH_RESET}              ${MAIN_WINDOW_KEYBOARD_CONTROLS}/Button[@AutomationId='ResetKeyboardInputs']
-${XPATH_LABEL_INPUT_UP}     ${MAIN_WINDOW_KEYBOARD_CONTROLS}/Text[@AutomationId='lblKeyboardKeyDown']
+${XPATH_LABEL_INPUT_UP}     ${MAIN_WINDOW_KEYBOARD_CONTROLS}/Text[@AutomationId='lblKeyboardKeyUp']
+${XPATH_LABEL_INPUT_DOWN}     ${MAIN_WINDOW_KEYBOARD_CONTROLS}/Text[@AutomationId='lblKeyboardKeyDown']
```
Comment (Nepitwin) [inline atests/Keyboard.robot:24]: Can you pls adjust spaces. Too many spaces are used. Pls set to arguments.

**Review by Nepitwin — CHANGES_REQUESTED** [review]
Pull request looks good. Only minimal changes must be done on robot tests. Afterwards this pr can be merged to main.

**Comment by Nepitwin** [comment]
@GeoGegl 

One thing do you forgot. Pls update changelog.md in root directory about parameters changes in keyboard keywords. Then all looks fine.

**Review by Nepitwin — APPROVED** [review]

**Review by noubar — APPROVED** [review]

**Pull request merged**